src/Stripe/Entities/StripeDiscountDelete.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Stripe {
```

Comment:
Fix this bracket to be on a new line. I need to get FxCop (or whatever is equivalent these days) for formatting guidelines.